src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -60,53 +59,43 @@ public static <T> Logger getLogger(Class<T> clazz) {
     }
 
     /**
-     * Adds the {@code consoleHandler} to the {@code logger}. <br>
-     * Creates the {@code consoleHandler} if it is null.
+     * Creates and returns a base logger with the default configuration with the name {@code BASE_LOGGER_NAME}
+     * and has a {@code ConsoleHandler} and a {@code FileHandler}.
```

Comment:
I commented on this earlier but the comment wasn't saved. Here it is again:
```suggestion
     * Creates and returns a base logger with the default configuration and containing a {@code ConsoleHandler}
     * and a {@code FileHandler}.
```